src/ui/actions/network.ts

```diff
@@ -126,3 +128,38 @@ export function selectAndFetchRequest(requestId: RequestId): UIThunkAction {
     });
   };
 }
+
+export function seekToRequestFrame(
+  requestId: RequestId,
+  frameId: string,
+  cx: Context
+): UIThunkAction {
+  return async (dispatch, getState, { ThreadFront }) => {
+    const state = getState();
+    const request = state.network.requests.find(request => request.id === requestId);
```

Comment:
a selector for this would be nice, especially since there is already another place where we fetch a request from redux.